### PR TITLE
fixed issue Misplaced pagination buttons in Users page #2683

### PR DESCRIPTION
**Pull request by saransh-dev**
#Fixed issue [#2683](https://github.com/apinf/platform/issues/2683)

### changes:-
client/accounts_admin.html
style/style.css

Closes: [#2683](https://github.com/apinf/platform/issues/2683)

### CODE REVIEW

File: client/accounts_admin.html [hunk 1]
```diff
@@ -42,7 +42,11 @@
     {{/each}}
     </tbody>
   </table>
-  <div class="form-inline">
+  <div class="text-center">  
+    {{> defaultBootstrapPaginator pagination=templatePagination limit=10 containerClass=""}}
```
Comment (marla-singer) [inline client/accounts_admin.html]: Add class `text-center` as value of attribute `containerClass` and remove upper `div` block. 
It allows the same design that you made but using a tool of pagination button

File: style/style.css [hunk 2]
```diff
@@ -26,4 +26,11 @@
 .pagination {
     margin-top: 0;
     margin-bottom: 0;
+}
+
+.users-items {
+    margin-top: -28px;
+}
+.users-items .items-on-page {
+    float: none;
```
Comment (marla-singer) [inline style/style.css]: Remove `float:left` from `.items-on-page` rule.

File: style/style.css [hunk 3]
```diff
@@ -26,4 +26,11 @@
 .pagination {
     margin-top: 0;
     margin-bottom: 0;
+}
+
+.users-items {
+    margin-top: -28px;
```
Comment (marla-singer) [inline style/style.css]: I found that you used it to place pagination button and text in one line. Really bad and dirty hack.

I propose not keeping these items in one line. Text about a count of users set under the table and pagination button will have a place in a new line.
joxi_screenshot_1504017774932

**Review by marla-singer — CHANGES_REQUESTED** [review]
Re-think about style. Keep the styling as simple as possible. After remaking remove useless styles

**Comment by saransh-dev** [comment]
@marla-singer  I have removed extra div and changed the code as you suggested. Please review.

**Review by marla-singer — APPROVED** [review]

**Pull request merged**